fusor-ember-cli/app/routes/register-nodes.js

```diff
@@ -13,6 +13,40 @@ export default Ember.Route.extend({
     controller.set('model', model);
     controller.set('showAlertMessage', false);
     var self = this;
+    var deployment_tasks = this.modelFor('deployment').get('deployment_tasks');
+
+    // TODO - use forEach with index rather than limiting to 5 tasks.
```

Comment:
note this comment. it can be reworked later. As I recall, the forEach with index is in Ember 1.12 and higher